api/src/pcapi/routes/backoffice_v3/filters.py

```diff
@@ -397,6 +397,22 @@ def format_website(website: str) -> str:
     return website
 
 
+def format_titelive_id_lectorat(id_lectorat: str) -> str:
+    match id_lectorat:
+        case "0":
+            return "Grand public"
```

Comment:
Juste pour être iso avec le reste
```suggestion
            return "grand public"
```